### DISCUSSION TITLE
关于在手动取消更新之后，Analyzer闪退的问题

### DISCUSSION THREAD

**Discussion by yeye233** [UmamusumeResponseAnalyzer/UmamusumeResponseAnalyzer#22] [Q&A] [answered]
由于一开始没有用梯子，更新很慢，所以关闭了UmamusumeResponseAnalyzer手动取消更新，再次打开的时候就发现Analyzer正常打开黑屏然后闪退的情况。之后删除.exe文件后从官网下载最新文件也同样黑屏闪退。由此想是不是有本地更新文件没有删除干净，就从用户的appdata内local文件内找到UmamusumeResponseAnalyzer文件夹进行删除。但是问题依旧存在，并且打开.exe文件后只在local相关文件夹内生成.config的一个空白文件，其他都没有了。想问下有没有解决的方法

**Comment by EtherealAO** [comment, marked as answer]
删掉%temp%目录下的latest-UmamusumeResponseAnalyzer.exe

**Reply by yeye233** [reply]
![issue](https://user-images.githubusercontent.com/60681746/168427650-bd1983dd-1a25-407c-9826-b471c42559de.png)
![issue2](https://user-images.githubusercontent.com/60681746/168427654-10024aa7-63d6-4584-91ce-43ad81d08db6.png)
谢谢大佬成功启动了，但是在更新时进行报错，然后重新闪退，这之后再启动也报错并闪退，重新安装并直接启动倒是没什么问题，但是只要一更新就又报错了

**Reply by yeye233** [reply]
![HT9E1)V{021~24XY$_N@$(X](https://user-images.githubusercontent.com/60681746/168428323-a53e6bbd-90f5-479e-bd4b-35e90897bd71.png)
群内的一些群友好像也出现了同样的问题

**Reply by EtherealAO** [reply]
国内用的更新服务器做了点改动，需要手动更新至[v1.8.7](https://github.com/EtherealAO/UmamusumeResponseAnalyzer/releases/tag/v1.8.7)，第二个reply好像是因为目录里含有空格，等有功夫了再看

**Reply by yeye233** [reply]
ok已经可以运行啦，谢谢大佬

**Reply by xk329** [reply]
大佬大佬，我把Temp里的那个exe删除了，但是更新的时候冒出来了不兼容64位windows是什么鬼

**Reply by xk329** [reply]
大佬强啊，手动更新完1.8.7之后就没有问题了，太强了
